### PR TITLE
CodeBlock: insert plugin button

### PR DESCRIPTION
**Pull request by lxgreen**
### :rocket: New Feature
- `common`
  - insert plugin button with `custom-block` type supported [allows to run custom add-block handler]
- `code-block`
  - insert-buttons added

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@
 
 ## [Unreleased]
 ### :rocket: New Feature
+- `common`
+  - insert plugin button with `custom-block` type supported [allows to run custom add-block handler]
```
Comment (bchmn) [inline CHANGELOG.md]: add link to PR

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,10 @@
...
+- `code-block`
+  - insert-buttons added
```
Comment (bchmn) [inline CHANGELOG.md]: add link to PR

File: packages/common/src/Base/createBaseInsertPluginButton.jsx [hunk 3]
```diff
@@ -9,6 +9,9 @@ import FileInput from '../Components/FileInput';
 import ToolbarButton from '../Components/ToolbarButton';
 import styles from '../../statics/styles/toolbar-button.scss';
 
+/**
+ * createBaseInsertPluginButton
+ */
```
Comment (bchmn) [inline packages/common/src/Base/createBaseInsertPluginButton.jsx]: necessary?

Comment (lxgreen) [reply packages/common/src/Base/createBaseInsertPluginButton.jsx]: good for search for imports and usages, since there's no function name

**Pull request merged**